commands/audit/audit_test.go

```diff
@@ -125,27 +297,15 @@ func TestAuditWithConfigProfile(t *testing.T) {
 			chdirCallback := clientTests.ChangeDirWithCallback(t, baseWd, tempDirPath)
 			defer chdirCallback()
 
+			auditParams.SetWorkingDirs([]string{tempDirPath}).SetIsRecursiveScan(true)
```

Comment:
why is it necessary here? wouldn't it detect it alone without setting the working dirs?
Also, defining working dirs and recursive scan is different from what our flow does

Reply:
No it's not, we do it outside the test scope here since you are calling `RunAudit` and not running the whole command.
This logic is the first thing in `Run()`:
![image](https://github.com/user-attachments/assets/5c2e615d-faed-4ecf-8f8d-4af851faa21c)
![image](https://github.com/user-attachments/assets/3504ff4f-0878-435c-ae14-020801676159)

If you are not setting the WD to be the temp dir, it will just scan the CWD that you are at the test